examples/ColumnGroupsResizeReorderExample.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright Schrodinger, LLC
+ */
+
+"use strict";
+
+import FakeObjectDataListStore from './helpers/FakeObjectDataListStore';
+import { TextCell } from './helpers/cells';
+import { Table, Column, ColumnGroup, Plugins } from 'fixed-data-table-2';
```

Comment:
What is the motivation for putting resize/reorder into a 'Plugins' namespace?  I understand that we want to decouple them from the core code, but I'm not sure it is clear to a new user coming to this code what it means to be a 'Plugin'.  Alternatively, resize/reorder could just be extendable components.

Reply:
One goal here is to separate out components that we intend for users to not make use of / replace with their own.  e.g. you could bring your own implementation of these to have different behaviors or UIs.

Reply:
Another goal would be that, maybe in the distant future, we'll provide this as a separate library.